skrf/vectorFitting.py

```diff
@@ -2349,7 +2349,7 @@ def get_new_subckt_identifier():
                     # add CCCS to generate the scattered current I_nj at port n
                     # control current is measured by the dummy voltage source at the transfer network Y_nj
                     # the scattered current is injected into the port (source positive connected to ground)
-                    f.write(f'F{n + 1}{j + 1} 0 a{n + 1} V{n + 1}{j + 1}'
+                    f.write(f'F{n + 1}{j + 1} 0 a{n + 1} V{n + 1}{j + 1} '
```

Comment:
Oh, it was missing the whitespace?

Reply:
yes, changed this by accident when applying lot of ruff suggestions I think.